findLongest.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * This method is to find the longest element in a Array
+ *
+ * @since 5.0.0
+ * @category Array
+ * @param {Array} array The array to inspect.
+ * @returns {string} Returns  longest element in a Array
+ * @example
+ *
+ * findLongest(["abc", 'abcd', "abcde"])
+ * // => "abcde"
+ */
+function findLongest(array) {
+  if (!Array.isArray(array)) {
+    throw new TypeError('Expected an Array')
+  }
+  if (!array.length) { return undefined }
+  const len = array.map(item => item.length)
+  const max = Math.max(...len)
+  const index = array.findIndex(item => item.length === max)
+  return array[index]
```

Comment:
Should we return `null` instead?

Reply:
i can't understand here, array is not empty

Reply:
🤔 This should've been put here by mistake. Resolving this one.

Reply:
return array[index] || null
 Is that right？